tools/mason/buildMason

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
+
+export CHPL_COMM=none
+export CHPL_LAUNCHER=none
+
+if command -v module > /dev/null 2>&1 ; then
+  echo "Unloading hugepages..."
+  module unload $(module list 2>&1 | \grep "craype-hugepage" | sed -e"s:[0-9]*)::g")
+fi
+
+if ! $chplBinDir/chpl mason.chpl ; then
```

Comment:
Do we check that `chpl` is in the `$PATH` elsewhere in the module build, and if not, would it be valuable to check it and give a separate error message for that case?

Reply:
Hmm, I'm not sure. Currently we're planning on people running 'make mason' from $CHPL_HOME, so I feel like it should be OK to use CHPL_BIN_DIR in this way.

Reply:
Ah, right. Catching this kind of error might be useful elsewhere (like a `make check` for `mason` or something), but definitely not necessary here.